policies/submissions.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Submitting Your Project To CODEVID-19
+---
+
+Read these carefully, otherwise we can't consider your team for prizing!
+
+### Deliverables
+
+1. **REQUIRED:** A link to your working app (the website URL, or a TestFlight/Google Play opt-in URL).
+2. **REQUIRED:** A link to a screenshot we can share with voters.
+3. **REQUIRED:** The [CODEVID-19 problem](rules.html#the-problems) you're addressing.
+4. **REQUIRED:** A short description and pitch for your app we can share on Twitter and social media (140 characters or less)
+5. **REQUIRED:** A long description and pitch for your app we can use to describe your app to voters and judges.
+6. **REQUIRED:** Instructions for interacting with your app (if not included in the app itself).
+7. **REQUIRED:** An email address that judges can use to follow up with the team should they have questions or concerns. This will not be shared with the public, or with voters.
+8. **REQUIRED:** A link to your project on FindCollabs
+9. **OPTIONAL:** For open-source projects, we encourage a link to the source code repository (e.g. GitHub, GitLab, Bitbucket, etc.).
+
```

Comment:
If you're judging impact, allowing them an opportunity to explicitly state the type of impact they believe they'll have will make it clearer to the judges and leaves less room for inferences/assumptions.

Reply:
Let's move this into a separate place, I'd prefer to ship the rules as-is. I think this is a better discussion in #wg-policy.

Reply:
Moved to #wg-policy